trin-core/src/portalnet/overlay.rs

```diff
@@ -374,7 +375,7 @@ impl<TContentKey: OverlayContentKey + Send> OverlayProtocol<TContentKey> {
     }
 
     /// Sends a request through the overlay service.
-    async fn send_overlay_request(
+    pub async fn send_overlay_request(
```

Comment:
Given that this method is doing some lower-level work, I would prefer that we keep this method private and instead create a public method to "initialize" a `FindContent` query.